app/views/coronavirus_landing_page/components/landing_page/_live_stream_section.html.erb

```diff
@@ -28,5 +32,12 @@
         </a>
       </li>
     <% end %>
+    <% if details.live_stream["popular_questions_link_visible"] == true %>
```

Comment:
I think we should be passing in `locals: { live_stream: details.live_stream }` to the live_stream_section partial [from the show page ](https://github.com/alphagov/collections/blob/8f3049d00f2f877fdbe7aaef4173d561efbbf8f6/app/views/coronavirus_landing_page/show.html.erb#L50) so we don't have to keep doing stuff like `details.live_stream["popular_questions_link_visible"]` we could just do `live_stream["popular_questions_link_visible"]`

Not related to your work, just spotted it as something to tidy up if you want to!

Reply:
Done ✅ 
Thanks for the feedback, that was a good suggestion. Made things tidier.